internal/databases/postgres/names_metadata.go

```diff
@@ -0,0 +1,15 @@
+package postgres
+
+import (
+	"fmt"
+)
+
+type PathsByNamesMetadata map[string][]string
```

Comment:
Lets remove the Metadata suffix and leave only the `PathByDbNames`? Since it is not a separate metadata but a part of another one.